zendesk/zd_ticket_comments_service.go

```diff
@@ -0,0 +1,106 @@
+package zendesk
+
+import (
+	"fmt"
+	"time"
+)
+
+// TicketComment represents a Zendesk Ticket Comment.
+//
+// Zendesk Core API docs: https://developer.zendesk.com/rest_api/docs/core/ticket_comments
+
+type TicketComment struct {
+	ID          int64        `json:"id,omitempty"`
+	Type        string       `json:"type,omitempty"`
+	Body        string       `json:"body,omitempty"`
+	HTMLBody    string       `json:"html_body,omitempty"`
+	PlainBody   string       `json:"plain_body,omitempty"`
+	Public      bool         `json:"public,omitempty"`
```

Comment:
Any reason to add **omitempty** in this **Public**?

Reply:
This is to omit the field of this particular struct to appear in json when it is empty.